citystreets/wsgi.py

```diff
@@ -11,6 +11,6 @@
 
 from django.core.wsgi import get_wsgi_application
 
-os.environ.setdefault("DJANGO_SETTINGS_MODULE", "citystreets.settings")
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "citystreets.settings.development")
```

Comment:
Sorry I am not sure if I completely understand this but let me ask anyway. It seems like the development settings are set as default. On the integration and production heroku instances, do we use the production settings from `citystreets/settings/production.py` or these development settings?

Reply:
You got it exactly.  The way it works is the majority of our settings from the original settings.py are in the common.py file. Any development (i.e. localhost) settings are in settings/development.py and production (heroku) settings are in settings/production.py.  Both development.py and production.py inherit common.py, and thus get all of those settings.  

It automatically detects whether it should be using development.py (the default) or production by the presence of a DJANGO_SETTINGS_MODULE environment variable. In the heroku apps that var will be set to 'citystreets.settings.production', which causes it to use the prod settings.  If that environment variable is not there though, it defaults to the dev settings.  It's a pretty neat little trick.

Reply:
I also wanted to ask about this line (and fell asleep without finishing the review haha)
Good point Muhammad!